packages/@react-aria/interactions/src/useFocusVisible.ts

```diff
@@ -79,7 +79,10 @@ function handlePointerEvent(e: PointerEvent | MouseEvent) {
 }
 
 function handleClickEvent(e: MouseEvent) {
-  if (isVirtualClick(e)) {
+  // Talkback double tap has detail === 1 for click events, so we check e.buttons
+  // instead
+  let isTalkBackClick = isAndroid() && e.buttons === 1;
+  if (isVirtualClick(e) || isTalkBackClick) {
```

Comment:
quick hack since `isVirtualClick` will return false for TalkBack double taps since TalkBack's event has `e.detail=1` when captured via a click listener instead of a pointerDown. Digging for a better way